lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -6043,20 +6043,22 @@ LogicalResult LayerBlockOp::verify() {
             if (getOperation()->isAncestor(definingOp))
               continue;
 
-          // Capture of a non-base type, e.g., reference, is allowed.
```

Comment:
nit: please keep the comments, those are helpful.

Reply:
OK! I felt like they were just repeating what the code said, but I can add it back.

Reply:
Fair enough... up to you. Reading this code for the first time, I found the comment helpful.